hack/ci-deploy.sh

```diff
@@ -37,6 +37,7 @@ mkdir -p "${deploy_dir}/"{operator,manager}
 cp ./deploy/manager/dev/*.yaml "${deploy_dir}/manager"
 cp ./deploy/operator/*.yaml "${deploy_dir}/operator"
 cp ./examples/common/cert-manager.yaml "${deploy_dir}/"
+cp ./hack/conntrack-cleaner-agent.yaml "${deploy_dir}/"
```

Comment:
this is platform specific while CI can run on multiple platform. I'd place it here https://github.com/scylladb/scylla-operator/blame/9d1ab0dce5c62a2a1607b0fb6d7aa8931ad6618c/.github/actions/setup-minikube/action.yaml